C/services/notification/notification_manager.cpp

```diff
@@ -1518,7 +1518,11 @@ bool NotificationManager::addDelivery(const ConfigCategory& config, const string
 			// Check and set registerIngest
 			if (deliver->ingestData())
 			{
-				deliver->registerIngest((void *)ingestCB, (void *)m_service);
+				AssetTrackInfo* trackingInfo  = new AssetTrackInfo;
```

Comment:
make sure that the trackinginfo is correctly deleted.

Reply:
It is passed to callback function, that's why can't be deleted otherwise callback function may crash